### PR TITLE
Add libaribb24 ARIB STD-B24 caption decoder

### PR DESCRIPTION
**Pull request by plainbanana**
See: https://github.com/jrottenberg/ffmpeg/issues/254
# Overview
Add support for ARIB STD-B24 caption decoder for MPEG-TS stream, which is supported by ffmpeg 4.2 or later. `libpng` and `libaribb24` are added as dependencies.
# Checklist
- [x] build Dockerfile
  - [x] All Dockerfile can be built successfully at local
  - [x] Some Dockerfile can be built successfully at local
- [x] update docs

### CODE REVIEW

File: docker-images/3.2/alpine38/Dockerfile [hunk 1]
```diff
@@ -453,6 +456,32 @@ RUN \
...
+## libaribb24
+RUN \
+        DIR=/tmp/b24 && \
+        mkdir -p ${DIR} && \
+        cd ${DIR} && \
+        curl -sLO https://github.com/nkoriyama/aribb24/archive/v${LIBARIBB24_VERSION}.tar.gz && \
+        echo ${LIBARIBB24_SHA256SUM} | sha256sum --check && \
+        tar -xz --strip-components=1 -f v${LIBARIBB24_VERSION}.tar.gz && \
+        autoreconf -fiv && \
+        ./configure CFLAGS="-I${PREFIX}/include -fPIC" --prefix="${PREFIX}" --enable-static --disable-shared && \
```
Comment (plainbanana) [inline docker-images/3.2/alpine38/Dockerfile:480]: [SHOULD] unnecessary option `--enable-static --disable-shared`

Comment (plainbanana) [reply docker-images/3.2/alpine38/Dockerfile]: Fixed by forced update

File: README.md [hunk 2]
```diff
@@ -118,18 +118,18 @@ Please use [Github issues](https://github.com/jrottenberg/ffmpeg/issues/new) to
 ## Test
 
 ```bash
-ffmpeg version 4.2.2 Copyright (c) 2000-2019 the FFmpeg developers
+ffmpeg version 4.3.1 Copyright (c) 2000-2020 the FFmpeg developers
```
Comment (plainbanana) [inline README.md:121]: [MUST] need to be updated

**Comment by plainbanana** [comment]
[IMO] About failed builds in CI

https://dev.azure.com/video-tools/ffmpeg/_build/results?buildId=348&view=logs&j=04093b27-7a2e-597d-49d4-a4254cb11b71&t=993abe88-2e41-52fb-1785-26251061261c&l=13086

It's failing the ZeroMQ test, which has nothing to do with this PR.

https://dev.azure.com/video-tools/ffmpeg/_build/results?buildId=348&view=logs&j=717d3127-f80b-5fe7-f6be-354a80de4837&t=41179017-2faf-5c8c-8bb9-8066b8bc75c8&l=12643

Failed to build because the sourceforge was down.

**Comment by jrottenberg** [comment]
it looks correct, let me check what happened on those failed builds.

**Pull request merged**